### DISCUSSION TITLE
Lists in Sphinx sidebar touch border

### DISCUSSION THREAD

**Discussion by medmunds** [pradyunsg/furo#130] [Potential Issue] [answered]
Lists inside a `.. sidebar::` seem to have the wrong indentation—the list indicators are up against the edge of the sidebar box:

<img src="https://user-images.githubusercontent.com/639984/114442213-2c39bd00-9b81-11eb-9da9-4c1cf0fd0b86.png" width="500">

I think the problem is that the CSS for [sidebar adds padding to its direct children](https://github.com/pradyunsg/furo/blob/2021.04.11.beta34/src/furo/assets/styles/content/_sidebar.sass#L13-L15), overriding any padding that's already there (e.g., the [padding lists need](https://github.com/pradyunsg/furo/blob/2021.04.11.beta34/src/furo/assets/styles/content/_lists.sass#L3) to leave space for indicators). I'm guessing this will create problems with any elements that include their own padding.

One solution might be to move the sidebar padding from `article .sidebar > *` up to `article .sidebar` itself (and then adjust the `.sidebar-title` margins and padding as necessary).

Source for image above:

```reStructuredText
Test
====

This page has a Sphinx sidebar.

.. sidebar:: Sidebar

   This sidebar has lists:

   * unordered
   * list

   Also:

   1. ordered
   2. list

This content is after the sidebar.
```

[Version 2021.04.11.beta34]

**Comment by pradyunsg** [comment, marked as answer]
Ah, nice catch! Fixed in 0165df3. I'll cut a release sometime this weekend.
